MonoGame.Framework.Content.Pipeline/Graphics/FontDescriptionStyle.cs

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
     public enum FontDescriptionStyle
     {
         /// <summary>
+        /// Normal text.
+        /// </summary>
+        Regular=0,
```

Comment:
Thanks @Tzenchor !

I ran into trouble last year getting the tests running under XNA (see https://github.com/mono/MonoGame/pull/3550).  Now with the next build server setup i'm going to try to get that working soon so that we always test both.